### ISSUE TITLE
Clicking on "terms and conditions" on sign-up should open up a new window

### ISSUE THREAD

**Issue by jonpmarino** [MapStory/mapstory-meta#1431]
Currently clicking it takes you away from the sign-up page

**Comment by Coop56** [comment]
So I went to go fix this really quick while I was thinking about another issue and the terms and conditions link is actually already setup to open in a new window.  So I'm not sure why it's not happening.  It's set to open in _blank like it should be.

**Comment by travislbrundage** [comment]
It's because of this code prevalent in a lot of our javascript files:

```
          // make sure that angular doesn't intercept the page links
          angular.element("a").prop("target", "_self");
```

It seems like a lot of links break without this, though I have no idea why...

I can hack a solution, but I feel like there needs to be another way to handle this.

**Comment by travislbrundage** [comment]
Closed by [PR#213](https://github.com/MapStory/mapstory-geonode/pull/213)
